semantic-conventions/.pylintrc

```diff
@@ -1,7 +1,7 @@
 [MESSAGES CONTROL]
 # TODO: We should actually fix the warnings instead of disabling them
 #   except for the stuff up to fixme, which is fine to ignore / defer to black. 
-disable = bad-whitespace, wrong-import-order, wrong-hanging-indentation, line-too-long, missing-docstring, unused-import, fixme, invalid-name, too-many-instance-attributes, too-many-locals, too-many-statements, too-many-branches, too-many-arguments, too-many-public-methods, too-few-public-methods, protected-access
```

Comment:
Why remove these? That stuff should be handled by black, not pylint.

Reply:
new pylint complains that it does not know them